packages/@ddes/postgres/lib/PostgresEventStore.ts

```diff
@@ -73,6 +73,47 @@ export class PostgresEventStore extends EventStore {
   }
 
   public async commit<TAggregateCommit extends AggregateCommit>(commit: TAggregateCommit) {
+    try {
+      await this.commitOne(this.pool, commit)
+    } catch (error: any) {
+      if (error.code === '23505') {
+        throw new VersionConflictError(commit)
+      }
+      throw error
+    }
+
+    return commit
+  }
+
+  public async commitInTransaction<TAggregateCommit extends AggregateCommit>(
+    commits: TAggregateCommit[]
+  ) {
+    const client = await this.pool.connect()
+
+    try {
+      await client.query('BEGIN')
+      for (const commit of commits) {
+        await this.commitOne(client, commit)
+      }
+      await client.query('COMMIT')
+    } catch (error: any) {
+      await client.query('ROLLBACK')
+      if (error.code === '23505') {
+        // TODO: Idenfity what commit caused the conflict
+        throw new VersionConflictError(commits[0])
+      }
+      throw error
+    } finally {
+      await client.release()
+    }
+
+    return commits
+  }
+
+  private async commitOne<TAggregateCommit extends AggregateCommit>(
```

Comment:
Nitpicky, but maybe call this `insertCommit` ?

Reply:
Addressed in d5ef9c4